launcher/minecraft/mod/Mod.cpp

```diff
@@ -147,28 +147,36 @@ void Mod::setStatus(ModStatus status)
     if (m_localDetails) {
         m_localDetails->status = status;
     } else {
-        m_temp_status = status;
+        if (!m_temp_status.get())
+            m_temp_status.reset(new ModStatus());
+
+        *m_temp_status = status;
```

Comment:
It's an enum class. I guess this doesn't need to be a pointer now that we don't copy construct `Mod` between places, but before that it was safer to since the copy construct corrupted this sometimes (why? idk o.O). I'll change it to a non-pointer though!